docs/terminology.md

```diff
@@ -0,0 +1,77 @@
+# Terminology
+
+This document describes terms and concepts often used within the APM ecossystem
+
+
+
+#### APM
+Application Performance Monitoring is the concept of profiling and monitoring systems
+
+#### Distributed Tracing
+Distributed tracing is a method for monitoring how applications interact with each other, especially in a microservice architecture.
+[Further reading](https://www.elastic.co/guide/en/apm/get-started/current/distributed-tracing.html)
+
+#### Instrumentation
+To be able to monitor an application it needs to be _instrumented_. Instrumentation can happen automatically (the Elastic APM agents instrument many things automatically) or manually.
```

Comment:
```suggestion
To be able to monitor an application it needs to be _instrumented_ by an APM Agent. 
Usually not everything is instrumented because that would incur a very large overhead. We are interested only in performance-sensitive activities, like database queries or web requests. Instrumentation can happen automatically (the Elastic APM Agents instrument many frameworks and databases automatically), or manually with a programmatic API.
```